onnxruntime/test/onnx/main.cc

```diff
@@ -224,9 +224,9 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
 
   // set log level based on number of verbosity options
   if (verbosity_option_count == 1) {
-    logging_level = ORT_LOGGING_LEVEL_INFO;
+    logging_level = ORT_LOGGING_LEVEL_WARNING;
   } else if (verbosity_option_count > 1) {
-    logging_level = ORT_LOGGING_LEVEL_VERBOSE;
+    logging_level = ORT_LOGGING_LEVEL_INFO;
```

Comment:
how about something like this for more flexibility:
`logging_level = ORT_LOGGING_LEVEL_ERROR - std::min(verbosity_option_count, ORT_LOGGING_LEVEL_ERROR)`
also i think verbosity_option_count can be unsigned

Reply:
Please check.

Reply:
looks good. i think we can still make verbosity_option_count unsigned to ensure it won't be negative, but it currently looks like it shouldn't be